README.md

```diff
@@ -55,3 +55,26 @@ After configuration is complete, running the link sharing is as simple as:
 ```
 $ linksharing run
 ```
+
+## Custom URL configuration and static site hosting*
+
+You can use your own domain for your linksharing and static site hosting with the following setup. 
+* We don't recommend utilizing this service for high traffic sites - due to security and cost-efficiency concerns - until we enable access to key shortening and page 
+caching.
+1. Share your READONLY file or directory via `uplink share --readonly --dns sj://<your path>`. 
+The `--dns` flag will print out the dns information that you will need.
+
+2. Create your CNAME with our linksharing common URL (https://link.tardigradeshare.io).
```

Comment:
this (and the associated screenshot) are not quite right - the CNAME value should just be `link.tardigradeshare.io.`, without the https://, with the trailing period. we're specifying a fully qualified domain name here (FQDN), not a protocol (https://), considering you could use other protocols with this same hostname. the trailing dot is weird but https://en.wikipedia.org/wiki/Fully_qualified_domain_name does indeed explain a bit more

Reply:
thanks for addressing that, I just had a TODO to look that up.

Reply:
done